lib/full-site-editing/template-parts.php

```diff
@@ -216,3 +216,21 @@ function gutenberg_do_block_template_part( $part ) {
 	}
 	echo do_blocks( $template_part->content );
 }
+
+/**
+ * Print the header template-part.
+ *
+ * @return void
+ */
+function gutenberg_do_block_template_part_header() {
```

Comment:
Maybe we can simplify to just `do_block_header` or `do_block_header_area`?

Reply:
Short and easier to digest. I like it. Done 👍

Reply:
I used the `do_block_header_area` because `do_block_header` would confuse a lot of people and they might think it's the header for a block (as opposed to a block header)